blivet/udev.py

```diff
@@ -143,8 +140,8 @@ def __is_blacklisted_blockdev(dev_name):
     if dev_name.startswith("ram") or dev_name.startswith("fd"):
         return True
 
-    if flags.installer_mode:
-        if any(re.search(expr, dev_name) for expr in INSTALLER_BLACKLIST):
+    if flags.blacklisted_devs:
+        if any(re.search(expr, dev_name) for expr in flags.blacklisted_devs):  # pylint: disable=not-an-iterable
```

Comment:
I still think this should be something like ``conf.blacklisted_devs``, having a list stored in something called ``flags`` is really counter-intuitive.